monitoring/README.md

```diff
@@ -86,7 +90,7 @@ pyroscope-alloy-0                                      2/2     Running   0
 a) WDS space:
 
 ```bash
-./configure-monitoring-wds.sh
+./configure-monitoring-wds.sh --host-cluster-context kind-kubeflex
```

Comment:
`--host-cluster-context` here means the same thing as `--hub-context` for the other script, right? If so then I recommend picking one and sticking with it. In KubeStellar we tend to talk about the "KubeFlex hosting cluster", while OCM/ACM talks about a "hub cluster" --- which corresponds to an ITS in KubeStellar.

Reply:
Yes!

Reply:
In the current rev of this PR, I only see "--hub-context" appearing in one place in this file, new line 69. So I am not understanding what you updated to match what.

I find the usage of the term "hub" in the context of KubeStellar to be confusing. In KubeStellar we use "core" to refer to all the central stuff and "KubeFlex hosting cluster" to refer to that specific thing. In plain English it would be reasonable to use "hub" to mean "the central stuff", but that positively conflicts with the technical term "hub" from OCM, which is important because KubeStellar uses OCM and a KubeStellar ITS is an OCM hub. So i see two plausible meanings of the word "hub" and I am rarely sure what anyone means by it. If you mean specifically the KubeFlex hosting cluster, then I strongly recommend using words like that. And where you mean a KubeStellar ITS, I strongly recommend using words like that.

Reply:
The flag `--hub-context` refers to the KubeFlex hosting cluster OR any other cluster that is hosting the core components of the KS control plane.

Reply:
What would be an example of an "other cluster that is hosting the core components of the KS control plane" but is not the KubeFlex hosting cluster?

Reply:
If I am not mistaken, there were some discussion about not having a tight coupling between KubeFlex and KS and to allow deployment of KS control plane in other cluster. Maybe these are just future plans. Then, I will replace the flag name `--hub-context` with `kubeflex-hosting-cluster-context`.

Reply:
If I understand correctly, this script only functions correctly if the context in question accesses a KubeFlex hosting cluster. For example, the use of `$wds-system` is specific to KubeFlex hosting. Unless you have a vision of how this script would work correctly in the general case, I recommend having the name of this flag be specific to KubeFlex hosting.